.github/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+<!--
+First, 🌠 thank you 🌠 for considering a contribution to Apollo!
+
+Some of this information is also included in the /CONTRIBUTING.md file at the
+root of this repository.  We suggest you read it!
+
+  https://github.com/apollographql/router/blob/HEAD/CONTRIBUTING.md
```

Comment:
I'm not sure what means HEAD here. In doubt I propose to use `main`.
```suggestion
  https://github.com/apollographql/router/blob/main/CONTRIBUTING.md
```

Reply:
I use `HEAD` to refer to the default branch which is `main` on this repository/gitremote.  I'm happy to change this to `main` if you'd like, but I opted to use this approach for referring to the default branch after the common pattern for default branch names changed to `main`.

Thoughts?

Reply:
Put another way, if the default branch of this repository changes, assuming nobody moves `CONTRIBUTING.md`, this link will not need to be updated.  Relatedly: https://git-scm.com/docs/git-remote#Documentation/git-remote.txt-emset-headem

cc @cecton

Reply:
I didn't know HEAD points to the default branch. So it's better with HEAD then!